SPECS/kernel-mshv/kernel-mshv.spec

```diff
@@ -237,8 +239,8 @@ ln -sf linux-%{uname_r}.cfg /boot/mariner.cfg
 %{_includedir}/perf/perf_dlfilter.h
 
 %changelog
-* Tue Jan 24 2023 Neha Agarwal <nehaagarwal@microsoft.com> - 5.15.86.mshv2-1
```

Comment:
Why overwrite changelog entry instead of adding a new one?

Reply:
Oops, I think git just merged it incorrectly. Thanks for catching that.